spinnaker-monitoring/spectator_client.py

```diff
@@ -33,6 +33,7 @@
 
 
 CONFIG_DIR = '/opt/spinnaker-monitoring/config'
+SOURCES_DIR = '/opt/spinnaker-monitoring/sources'
```

Comment:
I think you should do a `--sources_dir` flag, just like the `--config_dir` we specify in the Dockerfile